File: Interfaces/ITRFunctionService.cs
using WMS_WEBAPI.DTOs;
using System.Collections.Generic;
using System.Threading.Tasks;

namespace WMS_WEBAPI.Interfaces
{
    public interface ITRFunctionService
    {
        Task<ApiResponse<List<TransferOpenOrderHeaderDto>>> GetTransferOpenOrderHeaderAsync(string customerCode);
        Task<ApiResponse<List<TransferOpenOrderLineDto>>> GetTransferOpenOrderLineAsync(string siparisNoCsv);
        
    }
}

File: WebApi/Interfaces/ITRFunctionService.ts
import { ApiResponse } from '../Models/ApiResponse';
import { TransferOpenOrderHeaderDto, TransferOpenOrderLineDto } from '../Models/TRFunctionsDtos';

export interface ITRFunctionService {
  getTransferOpenOrderHeader(customerCode: string): Promise<ApiResponse<TransferOpenOrderHeaderDto[]>>;
  getTransferOpenOrderLine(siparisNoCsv: string): Promise<ApiResponse<TransferOpenOrderLineDto[]>>;
}